ACA-Py Build is failing

The ACA-Py build is failing as of a few days ago. The error log contains:

```
.
.
.
E: Unable to locate package libindy
E: Unable to locate package libnullpay
The command '/bin/sh -c apt-get update    && apt-get install -y git gnupg2 software-properties-common curl    && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88    && add-apt-repository 'deb https://repo.sovrin.org/sdk/deb bionic stable'    && apt-get update    && apt-get install -y libindy libnullpay' returned a non-zero code: 100
Docker image build failed.
Error: Process completed with exit code 1.
Run /home/runner/work/aries-agent-test-harness/aries-agent-test-harness/./test-harness/actions/run-test-harness-wo-reports/store-failed-env.sh -d acapy-main
Run jwalton/gh-docker-logs@v2
Found 8 containers...
```

--- a/aries-backchannels/acapy/Dockerfile.acapy
+++ b/aries-backchannels/acapy/Dockerfile.acapy
@@ -1,4 +1,4 @@
-FROM python:3.7-slim
+FROM python:3.7-slim-bullseye
 
 RUN apt-get update \
    && apt-get install -y git gnupg2 software-properties-common curl \

--- a/aries-backchannels/acapy/Dockerfile.acapy-main
+++ b/aries-backchannels/acapy/Dockerfile.acapy-main
@@ -1,4 +1,4 @@
-FROM python:3.7-slim
+FROM python:3.7-slim-bullseye
 
 RUN apt-get update \
    && apt-get install -y git gnupg2 software-properties-common curl \

